examples/alarm/alarm.py

```diff
@@ -0,0 +1,83 @@
+import uuid
+from datetime import datetime
+
+from nisystemlink.clients.alarm import AlarmClient
+from nisystemlink.clients.alarm.models import (
+    AlarmOrderBy,
+    AlarmTransitionType,
+    CreateAlarmTransition,
+    CreateOrUpdateAlarmRequest,
+    QueryAlarmsWithFilterRequest,
+    TransitionInclusionOption,
+)
+from nisystemlink.clients.core import HttpConfiguration
+
+# Setup the server configuration to point to your instance of SystemLink Enterprise
+server_configuration = HttpConfiguration(
+    server_uri="https://yourserver.yourcompany.com",
+    api_key="YourAPIKeyGeneratedFromSystemLink",
+)
+client = AlarmClient(configuration=server_configuration)
+
+# Create a unique alarm ID for this example
+# alarm_id is a user-defined identifier for this alarm
+alarm_id = f"example_alarm_{uuid.uuid1().hex}"
+
+# Create an alarm with a SET transition
+create_request = CreateOrUpdateAlarmRequest(
+    alarm_id=alarm_id,
+    transition=CreateAlarmTransition(
+        transition_type=AlarmTransitionType.SET,
+        occurred_at=datetime.now(),
+        severity_level=3,
```

Comment:
We should consider creating a set of `AlarmSeverityLevel` constants for the well-known severity levels:

- CLEAR=-1
- LOW=1
- MODERATE=2
- HIGH=3
- CRITICAL=4

The service supports levels greater than 4, but it's generally discouraged. In this case, it wouldn't be an enum, because we want to send an integer to the service.